src/env/root/mod.rs

```diff
@@ -400,6 +432,9 @@ struct Inner<'a> {
 
     call_targets: RefCell<HashSet<String>>,
     call_stack: RefCell<Vec<StackFrame>>,
+
+    #[cfg(feature = "debug")]
+    logger: RefCell<Box<dyn Fn(&str)>>,
```

Comment:
Is there a better way to describe this other than the `RefCell` of a `Box`'ed trait?

Reply:
You can do it as a generic type parameter, but this is probably fine.